SQL/0000-00-01-Modules.sql

```diff
@@ -54,5 +54,6 @@ INSERT INTO modules (Name, Active) VALUES ('dqt', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('electrophysiology_uploader', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('dataquery', 'Y');
 INSERT INTO modules (Name, Active) VALUES ('schedule_module', 'Y');
+INSERT INTO modules (Name, Active) VALUES ('redcap', 'Y');
```

Comment:
I think this should default to 'N'. The redcap integration is a specific ask of certain projects who can enable it through the module manager, it's not something that most new loris installs would want enabled by default.